### DISCUSSION TITLE
MAA自动把exe文件删除了

### DISCUSSION THREAD

**Discussion by ZenG009** [MaaAssistantArknights/MaaAssistantArknights#7648] [Q&A] [answered]
前几天开始我就遇到这个情况了，那时候没在意，以为是我自己不小心删掉了，刚刚又出现这个情况，我把公告关掉，然后maa也自动关了，然后我的exe文件就被删了，请问这是什么bug吗

**Comment by nanmingxwx** [comment]
安全中心报毒给删了

**Reply by ZenG009** [reply]
请问有什么解决方法吗，用好久了也没碰到这情况，这周开始遇到的

**Comment by orilights** [comment]
出现了类似的问题，昨天晚上任务还没完成就关机了，今天开电脑发现MAA的主程序直接没了，Windows自带杀毒已经关闭，保护记录里也没有，火绒也没记录，只能怀疑是自删的

**Reply by keytears** [reply]
一样用的火绒，系统里面也没杀毒记录火绒里面也没防护记录，已经自删了4次以上了。。。

**Reply by sswwd1** [reply]
一样的情况

**Comment by wonder041** [comment]
遇到了同样的问题. 
昨晚睡前还用着呢, 今天早上桌面快捷方式就失效了.
查了一下确实是windows给删掉了, 不知道为什么今天出的问题.
<img width="264" alt="捕获" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/19471830/0aa99e06-e52f-4eb1-9b8a-1897d39626a4">

**Comment by ABA2396** [comment, marked as answer]
#6662

**Comment by Sakamooto** [comment]
同样的问题 连着删三次了
